package.json

```diff
@@ -31,30 +32,34 @@
     "wouter": "2.5.1"
   },
   "devDependencies": {
-    "@types/async-retry": "1.4.2",
-    "@types/configstore": "4.0.0",
-    "@types/jest": "24.0.0",
     "@testing-library/jest-dom": "4.2.4",
     "@testing-library/react": "9.3.2",
     "@testing-library/user-event": "7.1.2",
     "@tsed/logger": "5.5.2",
+    "@types/async-retry": "1.4.2",
+    "@types/configstore": "4.0.0",
+    "@types/jest": "24.0.0",
+    "@types/lodash": "4.14.165",
     "async-retry": "1.3.1",
     "axios": "0.21.1",
     "configstore": "5.0.1",
+    "cross-env": "7.0.3",
+    "lodash": "4.17.21",
     "prettier": "2.1.1",
-    "react-scripts": "3.4.1",
+    "react-refresh": "0.10.0",
+    "react-scripts": "4.0.1",
     "ts-node": "9.1.1"
   },
   "scripts": {
-    "start": "yarn start:testnet",
-    "start:testnet": "cp ./config/testnet.json src/config.json && react-scripts start",
-    "start:sandbox": "cp ./config/sandbox-bootstrapped.json src/config.json && react-scripts start",
-    "start:mainnet": "cp ./config/mainnet.json src/config.json && react-scripts start",
+    "start": "pnpm start:testnet",
+    "start:testnet": "cross-env REACT_APP_NETWORK=testnet react-scripts start",
```

Comment:
After this change, I am having compilation issues with `src/config.json` not being available.

I see [below](https://github.com/tqtezos/minter/pull/313/files#diff-24e803dd0c81d1295b9f708a1c20a1d8bf930a961ef0f7cd9f1925156e449299R5) you're now referencing `REACT_APP_NETWORK` as a key on the `config` import:
```typescript
const cobj = (config as {[key: string]: any})[`${process.env.REACT_APP_NETWORK}`];
```
Perhaps there is a missing change which collocates all the environment configs under a single file and namespaces them by environment?

([src/config.json _is_ `gitignore`d](https://github.com/tqtezos/minter/blob/main/.gitignore#L64)).